extras/noVNC/websockify/__init__.py

```diff
@@ -1,2 +1 @@
-from websocket import *
-from websocketproxy import *
+from .websocketproxy import websockify_init
```

Comment:
I'd prefer to keep the API the same and only change the bare minimum.
```suggestion
from .websocket import *
from .websocketproxy import *
```